pupa/scrape/base.py

```diff
@@ -5,9 +5,10 @@
 from collections import defaultdict, OrderedDict
 
 import scrapelib
+from django.conf import settings
```

Comment:
Thinking about this, curious what @jpmckinney & @fgregg think about collapsing pupa's settings into Django settings like this.  

In general I'd like to move towards most of the config to be in environment variables, so it might not matter where we actually read them from.